article/app/services/dotcomponents/ArticlePicker.scala

```diff
@@ -90,9 +90,19 @@ object ArticlePageChecks {
     // See: https://github.com/guardian/dotcom-rendering/blob/master/packages/frontend/web/components/lib/ArticleRenderer.tsx
     def unsupportedElement(blockElement: BlockElement) =
       blockElement match {
-        case _: TextBlockElement  => false
-        case _: ImageBlockElement => false
-        case _                    => true
+        case _: TextBlockElement    => false
+        case _: ImageBlockElement   => false
+        case _: VideoBlockElement   => false
+        case _: GuVideoBlockElement => false
+        case _: EmbedBlockElement   => false
+        case ContentAtomBlockElement(_, atomtype) => {
+          // ContentAtomBlockElement was expanded to include atomtype.
+          // To support an atom type, just add it to supportedAtomTypes
+          val supportedAtomTypes =
+            List("media")
+          !supportedAtomTypes.contains(atomtype)
```

Comment:
I think this whole case can just be `case ContentAtomBlockElement(_, "media") => false`